uint8 leaf_id
uint16 addr
uint16 count
uint8 size
uint16[] values
---

uint8 exception_code
uint16 addr
uint16 count